### PR TITLE
Several fixes on links and images

### PR DESCRIPTION
**Pull request by M-DLB**
I've integrated most of the links as inline links (instead of references), except for some cases where it made sense to keep the references. I also fixed some of the images, but this will need to be reviewed.

### CODE REVIEW

**Comment by ijmitch** [comment]
LGTM

File: docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md [hunk 1]
```diff
@@ -113,7 +71,7 @@ Feature branches also need to relate back to the change request (or issue) from
 
 -   *hotfix/rel-2.0.1/fixMortgageCalculation* for a fix of the current production version that is running the *rel-2.0.1 release,*
 
--   *aiFraudDetection/IntroduceAIModelToMortgageCalculation* for a contribution to the development initiative for adopting AI technology for fraud detection.
+-   *epic/aiFraudDetection/IntroduceAIModelToMortgageCalcuation* for a contribution to a development initiative for adopting AI technology for fraud detection.
```
Comment (dennis-behm) [inline docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md:74]: @M-DLB why is this change being reverted back to a wrong naming convention pattern?

File: docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md [hunk 2]
```diff
@@ -34,68 +35,54 @@
...
 
 This approach can be compared to a trunk-based branching model that leverages feature branches. A similar workflow like outlined in this publication is also documented by Microsoft without giving it a name[^5].
 
 <!-- this paragraph would make an ideal 'annotation', callout or sidebar if writing for an SSG like Mkdocs. -->
-(You may also see reference to *topic* branches, for example [here](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows).
-Feature branches can be considered as one sub-type of *topic* branches with branches for *bug-fixes* being another sub-type.
-Both feature branches and bug-fix branches are expected to be relatively short-lived and ultimately to merge into a long-lived branch.)
+(You may also see reference to *topic* branches, for example [here](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows). Feature branches can be considered as one sub-type of *topic* branches with branches for *bug-fixes* being another sub-type. Both feature branches and bug-fix branches are expected to be relatively short-lived and ultimately to merge into a long-lived branch.)
```
Comment (dennis-behm) [inline docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md:53]: @M-DLB the above section still contains footnotes, which no longer exist. Is there propably a commit missing?

Comment (M-DLB) [reply docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md:53]: That part was also in the original document, so I didn't touch it.
But yes I agree it should not at least be between parenthesis. There is one comment (from @ijmitch I guess) in the original markdown file that says: "this paragraph would make an ideal 'annotation', callout or sidebar if writing for an SSG like Mkdocs.".
Maybe something @lauren-li can implement when migrating to the final version?

**Review by dennis-behm — APPROVED** [review]
Thanks @M-DLB . Final cosmetics will be done on the new solution websites.

File: docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md (hunk 2 above)
Comment (lauren-li) [reply docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md:53]: Following the result of our discussion the related [Slack thread](https://ibm-systems-z.slack.com/archives/C04JPQKF3UP/p1697016613178259), we will use Docusaurus's "Note" [admonition](https://docusaurus.io/docs/markdown-features/admonitions) in this case.

**Pull request merged**